cli/ioctl/cmd/action/action.go

```diff
@@ -136,7 +138,27 @@ func execute(contract string, amount *big.Int, bytecode []byte) (err error) {
 		signer,
 	)
 }
+func sendRaw(selp *iotextypes.Action) error {
+	conn, err := util.ConnectToEndpoint(config.ReadConfig.SecureConnect && !config.Insecure)
+	if err != nil {
+		return err
+	}
+	defer conn.Close()
+	cli := iotexapi.NewAPIServiceClient(conn)
+	ctx := context.Background()
 
+	request := &iotexapi.SendActionRequest{Action: selp}
+	if _, err = cli.SendAction(ctx, request); err != nil {
+		if sta, ok := status.FromError(err); ok {
+			return fmt.Errorf(sta.Message())
+		}
+		return err
+	}
+	shash := hash.Hash256b(byteutil.Must(proto.Marshal(selp)))
+	fmt.Println("Action has been sent to blockchain.")
+	fmt.Printf("Wait for several seconds and query this action by hash: %s\n", hex.EncodeToString(shash[:]))
```

Comment:
Try %x, which is hex format for bytes

Reply:
sure

Reply:
hex.EncodeToString  make it string already

Reply:
Yes, I mean with %x you don't need to call hex.EncodeToString